cmd/boilerplate.go

```diff
@@ -138,6 +166,30 @@ local config = require("` + pluginName + `.config")
 	}
 	defer mainLuaFile.Close()
 	mainLuaFile.WriteString(`local config = require("` + pluginName + `.config")`)
+
+	healthLua := filepath.Join(luaPath, "health.lua")
+	healthLuaFile, healthErr := os.Create(healthLua)
+	if healthErr != nil {
+		log.Fatal(healthErr)
+	}
+	defer healthLuaFile.Close()
+	healthLuaFile.WriteString(`local function checkhealth()
+   if vim.fn.has('nvim-0.5') then
+	  vim.fn['health#report_ok']('nightly installed')
+   else
+	  vim.fn['health#report_warn']('install neovim nightly')
```

Comment:
This would be a bit more future-proof if you mentionned '0.5' instead of nightly.

0.5 won't be a nightly forever !

Reply:
See 2694e2ad7841ba73380c5939af7b056344e18f12